src/App.js

```diff
@@ -1,25 +1,20 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Nav from './components/Navbar';
+import Book from './components/Book';
+import Categories from './components/Categories';
+
+const App = () => {  //eslint-disable-line
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="app">
+      <Nav />
+      <Routes>
+        <Route path="/Book" element={<Book />} />
```

Comment:
- [ ] In order to not show a blank page when visiting the site for the first time, Please make sure that the `Book` component is the main route. It will be like that:
```javascript
<Route path="/" element={<Book />} />
```